src/packages/BalancedJs/addresses.ts

```diff
@@ -16,6 +16,7 @@ const MAINNET_ADDRESSES = {
   bnusd: 'cx88fd7df7ddff82f7cc735c871dc519838cb235bb',
   baln: 'cxf61cd5a45dc9f91c15aa65831a30a90d59a09619',
   bwt: 'cxcfe9d1f83fa871e903008471cca786662437e58d',
+  fin: 'cx785d504f44b5d2c8dac04c5a1ecd75f18ee57d16',
```

Comment:
The variable only keeps the addresses related to the Balanced Network. remove fin token address.